### DISCUSSION TITLE
Bedrock

### DISCUSSION THREAD

**Discussion by lipki** [SummerFields/SummerFields#780] [Q&A]
Bedrock
You know, you don't know.
I never bothered with the bedrock version.
There are many reasons for this, the main reason and my allergy to Windows.
And then Chengzzi takes care of it and even if he takes the opportunity to sell it, that suits me well.
But hey, more and more people are using it, and will use it in the future.
It's high time I took it in hand.

But I don't know anything about this version, so I would like to know who among you (yes you) knows the bedrock version, and how resource packs work on this version.
I have just finally managed to launch the game on VirtualBox, I have never been so close.
But I might need some help.

Si vous parlez français, c'est mieux.

**Comment by tobyplowy** [comment]
Heyo @lipki I'd be interested in making this happen if I can sell it.

**Comment by lipki** [comment]
nope

**Comment by lipki** [comment]
We have already talked about it, it is not compatible with the license. We cannot sell the work of dozens of contributors, who have offered their work in a specific, non-commercial context.
